### DISCUSSION TITLE
ZY-M100-1 (_TZE204_7gclukjs)

### DISCUSSION THREAD

**Discussion by cmussoni** [Koenkk/zigbee2mqtt#22790] [Devices]
Hi everyone,
Has anyone already successfully integrated the ZY-M100-1 device (TS0601 TZE204_7gclukjs) into Home Assistant? I have several ZY-M100-1s integrated with no problems, but this one sees as unsupported. Thank you

**Comment by Koenkk** [comment]
How are the ones that are supported detected, can you provide a screenshot of the device -> about tab?

**Reply by cmussoni** [reply]
Hi, sorry for the delay.
The images are of the new device which is not recognized:

![a2](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/4c2e1b7e-117f-4a2d-b41c-e2e4df34d5b3)
![a3](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/b1aed3d6-c00f-4460-a6aa-ac6ca47841a4)
![a1](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/ec05717a-45e8-49c8-b38e-7a5d52c79fb2)

The below image is one of the working devices:

![b1](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/1582aa97-a7c1-47f7-8521-8ed376c5b853)

This is the main screen:
![tot](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/40414e13-60ec-4a40-9272-6e103b65e517)

 I have tried many times to delete it (0xa4c138516457010a) and reassociate it but to no avail.
Maybe it has a new firmware version?  Thank you.

**Comment by Koenkk** [comment]
Added! Will be supported in the Saturday release.

**Comment by cmussoni** [comment]
Hi, thanks for your work. I tried to reconfigure the device but I still get this error and the device does not return any light and motion values.
![1](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/61fdd132-784b-4ef1-ad66-e84b36c93688)
![3](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/6e69fddc-9448-4313-9cd7-eaa2b30be848)
![2](https://github.com/Koenkk/zigbee2mqtt/assets/10685435/c21b095f-2e2e-4454-b495-f8c7a37b8126)

**Comment by Koenkk** [comment]
Seems this device uses different datapoints, see [this guide](https://www.zigbee2mqtt.io/advanced/support-new-devices/03_find_tuya_data_points.html) on how to grab them.

**Comment by sygad** [comment]
Thanks for the update, i'll remove the external converter.

QQ
I remvoed and re-added 1 of the devices (I assume this step is neccessary), to ensure it would work without the external converter, it does, however, the icon is still showing the Zigbee icon, this isn't the same for other devices of a similar type

**Newest device showing zigbee icon**
![1](https://github.com/Koenkk/zigbee2mqtt/assets/1771605/dff37c62-1065-4306-bb02-328be266e35f)
![2](https://github.com/Koenkk/zigbee2mqtt/assets/1771605/afe1e577-04bf-4529-8547-8dfb6067f929)

**Older devices showing 24G icon**
![3](https://github.com/Koenkk/zigbee2mqtt/assets/1771605/baed500e-a743-4f6b-8ad8-f52cd29f9ef5)
![4](https://github.com/Koenkk/zigbee2mqtt/assets/1771605/9c956a93-4409-4faa-8dd8-a4c4ec75c731)
